src/js/tabs/firmware_flasher.js

```diff
@@ -219,27 +216,23 @@ TABS.firmware_flasher.initialize = function (callback) {
             loadUnifiedBuilds(releases);
         };
 
-        function checkOneVersionForUnification(version) {
+        function supportsUnifiedTargets(version) {
             return semver.gte(version.split(' ')[0], '4.1.0-RC1');
```

Comment:
Not your code, but to me is a little strange continue supporting the RC versions and only starting with the RC1. But I suppose is needed to not break things...

Reply:
I can check tomorrow but at the time it seemed best. 4.1.0 is newer than 4.1.0-RC1 so if we specified 4.1.0 we wouldn't have had unified targets with any RC. Specifying something like gte 4.0.9999 would have worked but then it's not clear to the humans as to what is going on.

Reply:
@McGiverGim: This is not so much about 'supporting 4.1.0-RC1', but about making sure that we are not trying to use custom defaults with any (released) versions that do not support this. And 4.1.0-RC1 is the first released version to support custom defaults.